packages/web3-net/src/index.ts

```diff
@@ -15,6 +15,36 @@ You should have received a copy of the GNU Lesser General Public License
 along with web3.js.  If not, see <http://www.gnu.org/licenses/>.
 */
 
+/**
+ * The web3-net package allows you to interact with an Ethereum node’s network properties.
+ *
+ * ```ts
+ * import Net from 'web3-net';
+ *
+ * const net = new Net(Net.givenProvider || 'ws://some.local-or-remote.node:8546');
+ * // or using the web3 umbrella package
+ * import Web3 from 'web3';
+ * const web3 = new Web3(Web3.givenProvider || 'ws://some.local-or-remote.node:8546');
+ *
+ * // -> web3.eth.net
+ *
+ * // get the ID of the network
+ * web3.eth.net.getId();
```

Comment:
I think it is important to indicate here, and in similar places, that the call is ´async´. And this will also make the code snippet more accurate. Or what do you think?

```suggestion
 * await web3.eth.net.getId();
```

Reply:
completely agree, thanks for this catch :) @Muhammad-Altabba